backend/index.js

```diff
@@ -268,3 +379,52 @@ app.get('/patchnotes/overwatch/:year/:month/comments/:commentId/replies', async
         res.status(500).json({ error: 'Failed to fetch replies' });
     }
 });
+
+// associations
```

Comment:
While adding this comment does help to break up the structure of the file, this comment as is is kind of useless -- we can see that this section refers to associations just by the names of the functions. Can you add more descriptive language as to what these associations are?

Reply:
+1